Update Mocha to 2.1.0

backspace opened the pull request:
Hey, Mocha 2.1.0 features this [small fix](https://github.com/mochajs/mocha/commit/221e0360e23a41e4b8583a22a7a5f3a3aa5f55cb) that restores the ability to click the name of a test to run it individually, which obviates the minor-but-persistent hassle of re-adding “tests” to the beginning of the pathname when I need to focus on a test.

I was specifying 2.1.0 in `bower.json` and that worked okay locally, but Heroku and Travis builds were failing, so I figured I’d make this fork.

This update might not be useful for everyone, feel free to close if it seems unnecessary.

bjornstar commented:
Any chance of revisiting this? I'd like to use mocha 2.1.0 as well. I tried updating my bower.json file and don't have any problems running `ember test`

backspace commented:
@bjornstar, when I changed `bower.json` to point to 2.1.0 it worked fine, it was when I switched `ember-mocha` to `backspace/ember-mocha#update-mocha` that it was hanging.

I hadn’t tried it since then, but now when I do, I get this:

    Path or pattern "bower_components/ember-mocha/ember-mocha.amd.js" did not match any files

I think I was on Ember CLI 0.1.4 at the time and now I’m on 0.1.7, so maybe something got fixed to print out an error when an expected file is missing. It looks like `ember-mocha.amd.js` is built when the package is installed by Bower, but that build process isn’t happening when I use my fork, for whatever reason. This is beyond my understanding.

bjornstar commented:
Ah yes, I get completely different behavior from bower when I install from a repo vs installing the public `ember-mocha`

Putting these three in my bower.json file illustrates the problem very well.
```
    "ember-mocha": "~0.2.0",
    "ember-mocha-backspace": "backspace/ember-mocha#update-mocha",
    "ember-mocha-switchfly": "switchfly/ember-mocha#0.2.1",
```

The build process does not run when you install from a repo.

You'd have to run the build scripts yourself.

bjornstar commented:
Can you reopen this? Or should I make a PR that does the same thing?
